### ISSUE TITLE
Play/Pause Button Missing From Latest Release (0.12)

### ISSUE THREAD

**Issue by Firedingo** [jbonjean/beluga-player#12]
I got the latest release to find the play/pause button is missing from the interface which is kind of a problem.

I've attached a screenshot to show. It seems there's a new volume control and skip still but the play/pause button has magically vanished.

![image](https://user-images.githubusercontent.com/47981280/75348253-926c6d80-58f6-11ea-9ed9-737eacd1bf5e.png)

I tried resizing the window but that had no effect.

Also the audio unmutes when a new song is loaded. I suppose I should make a new post for that?

**Comment by jbonjean** [comment]
Yes, the pause button had some bad side effects, like tracks or session token expired on resume and on top of that I felt it was against the radio streaming concept (there is no pause in the official UI). So I decided to change it to for a mute button, which makes more sense to me. I was aware of the audio unmute on new song, this is a bug I will fix.

To be honest, I wasn't expecting somebody else was using this software, that's why I just did the change without thinking it would impact somebody else.
